src/simplified-account-routes.js

```diff
@@ -323,6 +322,14 @@ simplifiedAccount.post(
   servicesController.agreements.cancel.post
 )
 
+// transactions
+simplifiedAccount.get(
+  paths.simplifiedAccount.transactions.index,
+  experimentalFeature,
+  permission('transactions:read'),
+  servicesController.transactions.list.get
+)
```

Comment:
there's already an entry for this route on line 288 - it does need the `experimentalFeature` middleware there though

Reply:
if you could add the middleware for the transaction details route on line 294 as well that would be great